api/app/Generators/CandidateProfileCsv.php

```diff
@@ -0,0 +1,219 @@
+<?php
+
+namespace App\Generators;
+
+use App\Enums\PositionDuration;
+use App\Models\Pool;
+use App\Models\PoolCandidate;
+use Exception;
+
+class CandidateProfileCsv extends CsvGenerator
+{
+    protected array $ids;
+
+    protected string $lang = 'en';
+
+    protected array $headers = [
+        'Status',
+        'Category',
+        'Availability',
+        'Notes',
+        'Current province',
+        'Date received',
+        'Expiry date',
+        'Archival date',
+        'First name',
+        'Last name',
+        'Email',
+        'Preferred communication language',
+        'Preferred spoken interview language',
+        'Preferred written exam language',
+        'Current city',
+        'Armed forces status',
+        'Citizenship',
+        'Bilingual evaluation',
+        'Reading level',
+        'Writing level',
+        'Oral interaction level',
+        'Estimated language ability',
+        'Government employee',
+        'Department',
+        'Employee type',
+        'Current classification',
+        'Priority entitlement',
+        'Priority number',
+        'Location preferences',
+        'Location exemptions',
+        'Accept temporary',
+        'Accepted operational requirements',
+        'Woman',
+        'Indigenous',
+        'Visible minority',
+        'Disability',
+        'Education requirement',
+        'Education requirement experiences',
+    ];
+
+    protected array $questionIds = [];
+
+    protected array $skillIds = [];
+
+    public function __construct(array $ids, ?string $lang = 'en')
+    {
+        $this->ids = $ids;
+        $this->lang = $lang;
+
+        parent::__construct();
+    }
+
+    public function generate()
+    {
+        $candidates = PoolCandidate::with([
+            'educationRequirementAwardExperiences',
```

Comment:
On L145 and L191 you're using just educationRequirementExperiences and it looks like you're not using and of the  user->*experiences at all.

Reply:
ugh, I gotta turn off copilot :face_exhaling: I fixed this like 3 times but every time I went to change this it kept auto-completing to this every time :angry: and since it is so close I rarely ever noticed it :cry:

Reply:
Also noticed we removed the bilingual eval option so I fixed(?) that as well.

[fix attribute access errors](https://github.com/GCTC-NTGC/gc-digital-talent/pull/9988/commits/8df48157d266a985e857c5f49ec93e290f1324f7)

Reply:
This is working well now so I'll approve.  I still think the user->*experiences relationships don't need to be `with`'d at all.  And you have userSkills `with`'d twice.

Reply:
Oops, did not notice that :sweat_smile: I also didn't realize the education experience eager load wasn't doing anything. It was complaining before your merged all the morphs so I guess I just assumed it was still needed after :woman_shrugging: Anway! I removed both in [remove unnecessary eager loads](https://github.com/GCTC-NTGC/gc-digital-talent/pull/9988/commits/0622e3c7fca5de6a9566996a99a398174cdf166d) and it is still working :+1: